config/initializers/civicrm_verification_workflow.rb

```diff
@@ -4,7 +4,6 @@
   workflow.form = "Decidim::Verifications::Civicrm"
 
   workflow.options do |options|
-    options.attribute :role, type: :enum, choices: -> { Decidim::Civicrm::Api::User::ROLES.values.map(&:to_s) }
```

Comment:
Actually, I messed up writing the issue 🙏🏽  this line should be left as-is, so that admins can restrict actions to users with the `:inscribed` role (to prevent regular `:interested` users from doing certain things)

Reply:
But with the other commit we make sure that there are not :interested roles, right? I thought that we were not going to have any :interested users but only :inscribed

Reply:
The thing is that these `options.attribute :role` will be used in the admin to restrict actions for users with the `inscribed` role:

![image](https://user-images.githubusercontent.com/8806781/98132737-0cf58000-1ebd-11eb-9ee9-a276891f6fd6.png)